integrations/airflow/tests/integration/docker/up.sh

```diff
@@ -20,8 +20,10 @@ set -e
 project_root=$(git rev-parse --show-toplevel)
 cd "${project_root}"/integrations/airflow/tests/integration
 
-REV=$(git rev-parse HEAD)
-MARQUEZ_AIRFLOW_LIB_WITH_REV="git+git://github.com/MarquezProject/marquez.git@${REV}#egg=marquez_airflow&subdirectory=integrations/airflow"
+GIT_URL=$(git config --get remote.origin.url \
```

Comment:
Note, we have to modify the link to use `https`, see https://github.com/pypa/pip/issues/7554